### ISSUE TITLE
A11y_.NET Core_WinForms_PrintingTesting_PrintingTesting_Keyboard: Keyboard focus is not going to the "View your page" "Pane".

### ISSUE THREAD

**Issue by HindujaArugula** [dotnet/winforms#9275]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No response

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG2.1.1;#DesktopApp;#Win11;#NonBenchmark;#Keyboard;

### Actual:
Keyboard focus is not going to the "View your page" "Pane".

### Expected:
Keyboard focus should go to the "View your page" "Pane".

### User Impact:
Keyboard only users will face difficulties if the keyboard focus does not to the "View your page" pane.

### Steps to reproduce

1. Install and Open "Winforms" App.
2. "Main Form" screen would appear.
3. TAB to "Printing_Controls:Testing the controls under the Printings Tab" and press ENTER key.
4. "Printing Testing" screen appears.
5. Now observe that Keyboard focus is not going to the "View your page" "Pane".

### Attachment:
![Keyboard1](https://github.com/dotnet/winforms/assets/125462007/c0daa196-d1a1-4523-85d8-1675906de30f)

**Comment by merriemcgaw** [comment]
@dmitrii-drobotov @Tanya-Solyanik are we able to get the pane selected? This is not something that is editable. Would perhaps different settings on Narrator make it happen?

**Comment by dmitrii-drobotov** [comment]
Yeah, I found these issues: https://github.com/dotnet/winforms/issues/3660 and https://github.com/dotnet/winforms/issues/3678. From what I can tell, it's intended that this pane is not selectable.

**Comment by dmitrii-drobotov** [comment]
>  Are there any keyboard shortcuts that control scroll bars? Scroll Lock key does not work for me in winforms apps.

Print preview can be scrolled by Ctrl+Arrows, but it needs to have `TabStop = true`.
![A gif showing that PrintPreviewControl content can be scrolled by Ctrl+Arrows](https://github.com/dotnet/winforms/assets/102954094/ca147d5d-50cb-4723-a9d7-a2685c6e19a7)

**Comment by Tanya-Solyanik** [comment]
>Print preview can be scrolled by Ctrl+Arrows, but it needs to have TabStop = true.

Yeah, I think it was a mistake to remove TabStop there.

**Comment by merriemcgaw** [comment]
OK, let's get that in.

**Comment by Ashley-Li** [comment]
Verified this on .NET 8.0 latest build: 8.0.100-rc.1.23402.12, it was fixed. Now keyboard focus is going to the "View your page" "Pane".

https://github.com/dotnet/winforms/assets/56664234/68733d36-4187-4b8f-9b77-01757e7ebc42

**Comment by HindujaArugula** [comment]
@merriemcgaw Could you please provide the latest environment to verify the issue.
PendingEnviornment; Regressed:08-16-23;

**Comment by Tanya-Solyanik** [comment]
@Ashley-Li , @Olina-Zhang Could you please help with the current test environment?

**Comment by Olina-Zhang** [comment]
Here is the testing app: \\\mlangfs1\public\Winforms\CoreTest\AccessibilityTestingApp\CoreApplication\CoreSDK8.0_Test_app\Accessibility_Core_8.0_AppExe.zip
and required .Net 8.0 SDK: \\\mlangfs1\public\Winforms\CoreTest\AccessibilityTestingApp\CoreApplication\CoreSDK8.0_Test_app\dotnet-sdk-8.0.100-rc.2.23417.14-win-x64.exe

**Comment by Amy-Li03** [comment]
Verified this on .NET 8.0 RC1 test pass build: 8.0.100-rc.1.23421.2, it was fixed, test result is same as [above video](https://github.com/dotnet/winforms/issues/9275#issuecomment-1663529676).

**Comment by HindujaArugula** [comment]
Verified the issue in latest testing app i.e., \\mlangfs1\public\Winforms\CoreTest\AccessibilityTestingApp\CoreApplication\CoreSDK8.0_Test_app\Accessibility_Core_8.0_AppExe.zip.
Issue is fixed and does not repro. Hence closing the bug.
Please find closing attachment.
\\mlangfs1\public\Winforms\CoreTest\AccessibilityTestingApp\CoreApplication\CoreSDK8.0_Test_app\Accessibility_Core_8.0_AppExe.zip
#Closed; #Regressed:08-24-23
![Keyboard focus_Fixed](https://github.com/dotnet/winforms/assets/125462007/f5f0ae00-e12b-409d-9c39-5af96250b643)
